frontend/public/components/RBAC/bindings.jsx

```diff
@@ -131,7 +131,7 @@ const Row = ({obj: binding}) => <ResourceRow obj={binding}>
   <OverflowYFade className="col-md-2 col-sm-4 col-xs-6 co-break-word">
     {binding.metadata.namespace ? <ResourceLink kind="Namespace" name={binding.metadata.namespace} /> : 'all'}
   </OverflowYFade>
-  <div className="co-kebab-wrapper">
+  <div className="co-kebab-wrapper dropdown-kebab-pf">
```

Comment:
Why do we need this wrapper class around every usage of kebab components? Seems pretty easy to forget...

Reply:
Yeah, it's a good question. We need something that does the positioning.

```css
.co-kebab-wrapper {
  position: absolute;
  right: 0;
  right: 15px;
  top: 7px;
}
```

We could use `.co-resource-list__item .dropdown-kebab-pf`. That has the drawback of not working in other tables, though.

Reply:
It's currently only in list pages